parm/config/config.defaults.s2sw

```diff
@@ -15,14 +15,16 @@ FHOUT_HF_GFS=-1
 IALB=2
 IEMS=2
 DO_CA="YES"
-min_seaice="1e-11"
+min_seaice="1e-6"
+use_cice_alb=".true."
+export iopt_sfc="3"
```

Comment:
Some of the other variables are exported.  I did this because it wasn't over-writing a value in another config file, so I wanted to make sure it'd be exported so it would be used when creating the input.nml file.  I can try to remove the export statement and see if this still gets set appropriately though.